### PR TITLE
add common aliases for languages

### LINKED ISSUE #69: !lib aliasing
using !lib should accept common aliases, like .net, c#, js, ts

### PR DESCRIPTION
**Pull request by stelar7**
fixes #69

### CODE REVIEW

File: settings/shared_settings.json [hunk 1]
```diff
@@ -52,7 +52,19 @@
         "languageList": "I have libraries for one of the following languages: {languages}\nChoose one you like and reply with `!libs <your language>`.",
         "githubErrorList": "I tried to get the languages from Github, but Github replied with status code ",
         "githubErrorLanguage": "I tried to get the language from Github, but Github replied with status code ",
-        "baseURL": "https://api.github.com/repos/WxWatch/riot-api-libraries/contents/libraries/"
+        "baseURL": "https://api.github.com/repos/WxWatch/riot-api-libraries/contents/libraries/",
+        "aliases": {
+            "javascript": [
+                "js",
+                "ts",
+                "typescript"
```
Comment (Querijn) [inline settings/shared_settings.json]: 🤔

Comment (Querijn) [reply settings/shared_settings.json]: Would not put this one in the js aliases

File: settings/shared_settings.json [hunk 2]
```diff
@@ -52,7 +52,19 @@
...
+            ],
+            "c-sharp": [
+                ".net",
```
Comment (Querijn) [inline settings/shared_settings.json]: Nor this one, they're just not the same

File: src/RiotAPILibraries.ts [hunk 3]
```diff
@@ -129,9 +129,21 @@ export default class RiotAPILibraries {
...
 
-    private async getListForLanguage(message: Discord.Message, language: string) {
+    private async getListForLanguage(message: Discord.Message, language: string): Promise<void> {
         const response = await fetch(this.settings.riotApiLibraries.baseURL + language);
+
         if (response.status !== 200) {
+            const aliases = this.settings.riotApiLibraries.aliases;
+            const keys = Object.keys(aliases);
+
+            for (const index in keys) {
```
Comment (molenzwiebel) [inline src/RiotAPILibraries.ts]: `for (const [, key] in Object.entries(aliases))`

Comment (stelar7) [reply src/RiotAPILibraries.ts]: It doesnt like that 🤔 

`The left-hand side of a 'for...in' statement cannot be a destructuring pattern.`

Comment (stelar7) [reply src/RiotAPILibraries.ts]: I cant get VSC to accept Object.entries in es6, so we would need to update to es2017? 🤔

Comment (Querijn) [reply src/RiotAPILibraries.ts]: That is fine

**Pull request merged**